configure.ac

```diff
@@ -1431,6 +1431,36 @@ AC_SUBST(OPENTHREAD_ENABLE_DHCP6_SERVER)
 AM_CONDITIONAL([OPENTHREAD_ENABLE_DHCP6_SERVER], [test "${enable_dhcp6_server}" = "yes"])
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_DHCP6_SERVER],[${OPENTHREAD_ENABLE_DHCP6_SERVER}],[Define to 1 if you want to enable DHCPv6 Server])
 
+#
+# IPv6 Fragmentation
+#
+
+AC_ARG_ENABLE(ip6_fragmentation,
```

Comment:
Given the support of #3964, can we move this build option to [src/core/openthread-core-default-config.h](https://github.com/openthread/openthread/blob/master/src/core/openthread-core-default-config.h)?

Reply:
Yeah, I guess so.

Reply:
Moved to the `src/core/openthread-core-default-config.h` and renamed to `OPENTHREAD_CONFIG_ENABLE_IP6_FRAGMENTATION`.